### DISCUSSION TITLE
Allow capturing screenshot on multiple Android devices similar to iOS

### DISCUSSION THREAD

**Discussion by xming13** [fastlane/fastlane#17053] [Q&A]
May I know if capturing screenshot on multiple Android devices is supported?
e.g. specify a list of emulator ids in `Screengrabfile` and running `fastlane action screengrab` will automatically capture screenshot on each emulator. Similar to how it works in iOS.

Reference: same issue without conclusion: https://github.com/fastlane/fastlane/issues/14172

**Comment by sp71** [comment]
It does not currently exist. Here is a workaround I found [online](https://github.com/strooooke/quickfit/blob/09a7d40038346e3238c11b09338743195d62dddc/make_screenshots.sh) from this [article](https://lambdasoup.com/post/automate_screenshots_1/) where they create a script to start up the emulator and loop through the device list to run screengrab
